injected/entry-points/apple.js

```diff
@@ -8,7 +8,34 @@ import { WebkitMessagingConfig, TestTransportConfig } from '../../messaging/inde
 
 function initCode() {
     // @ts-expect-error https://app.asana.com/0/1201614831475344/1203979574128023/f
-    const processedConfig = processConfig($CONTENT_SCOPE$, $USER_UNPROTECTED_DOMAINS$, $USER_PREFERENCES$);
+    const config = $CONTENT_SCOPE$;
+
+    config.features.messageBridge = {
+        exceptions: [],
+        state: 'enabled',
+        settings: {
+            allowed: [],
+            domains: [
+                {
+                    domain: 'ddgshane.ngrok.io',
+                    patchSettings: [
+                        {
+                            op: 'replace',
+                            path: '/allowed',
+                            value: ['aiChat', 'duckPlayer'],
+                        },
+                    ],
+                },
+            ],
+        },
+    };
```

Comment:
An example of how the config might look for this feature. Open to ideas, but mainly I want to stick within the feature set already supported by CSS (the example here is already supported for domain restrictions)

Reply:
I'm not a fan of patches in the base config.. Have you considered mapping feature -> domains instead of domain -> features? I assume that the number of features will always be limited, but the number of domains may grow, so it could be a more compact structure:

```javascript
"messageBridge": {
  "exceptions": [],
  "state": "enabled",
  "settings": {
    "features": {
      // we could type keys here to be a subset of the top-level "features"
      "aiChat": {
        "allowedOn": ["ddgshane.ngrok.io", "duckduckgo.com"]
      },
      "duckPlayer": {
        "allowedOn": ["ddgshane.ngrok.io", "duck.ai"]
      }
    }
  }
}
```

Reply:
@muodov - I chose that format because it's already fully supports domain-restrictions, meaning my code is just this: `this.getFeatureSettings('allowed')`. Wouldn't this format mean I need to do the checks again?

@jonathanKingston  is there a way to have best of both worlds? FWIW I honestly have zero preference on this, and I very rarely work with config stuff - so if either of you have strong preferences then I'm happy to see them in action.

Reply:
> is there a way to have best of both worlds?

I think we did support a domains method once, can't remember.
Why invent a new pattern though?

Reply:
will move to an asana discussion

Reply:
`matchDomainFeatureSetting` is the alternate tbh.

Reply:
yeah this works 

```
    config.features.messageBridge = {
        exceptions: [],
        state: 'enabled',
        settings: {
            allowed: [
                {
                    domain: 'ddgshane.ngrok.io',
                    features: ['aiChat', 'duckPlayer'],
                },
            ],
        },
    };
```